config/database.yml

```diff
@@ -24,6 +24,12 @@ default: &default
 development:
   <<: *default
   database: TasteDiary_development
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: 2599
+  host: localhost
+  port: 5432
```

Comment:
@naanmohammed Maybe this configuration is specific to your postgres database on your pc? 🤔